lib/changes.js

```diff
@@ -53,7 +53,7 @@ export function addNodes(selectorObserver, changes, nodes) {
   for (let i = 0; i < nodes.length; i++) {
     const node = nodes[i]
 
-    if ('matches' in node) {
+    if (supportsSelectorMatching(node)) {
```

Comment:
Checking if `node` is an `Element` might be acceptable here too. Let me do some testing.

Reply:
Ah, `document` might be the edge case in some of these code paths. For an example, document implements `querySelectorAll` but is not an `Element`.